collector/prom-native-clickhouse/README.md

```diff
@@ -0,0 +1,84 @@
+# Ingest Clickhouse metrics using the OpenTelemetry Collector
+
+## Overview
+
+ Clickhouse natively exposes a Prometheus endpoint and the OpenTelemetry Collector has a [Prometheus receiver][otel-prom-receiver] that can be used to scrape its Prometheus endpoint. This directory contains an example showing how to configure Clickhouse and the Collector to send metrics to Lightstep Observability.
+
+## Prerequisites
+
+* Docker
+* Docker Compose
+* A Lightstep Observability [access token][ls-docs-access-token]
+
+## How to run the example
+
+* Export your Lightstep access token
+  ```
+  export LS_ACCESS_TOKEN=<YOUR_TOKEN>
+  ```
+* Run the docker compose example
+  ```
+  docker-compose up -d
+  ```
+* Run clickhouse client
+  ```
+  make run-client
+  ```
+  * Test DB
+  ```
+  CREATE DATABASE IF NOT EXISTS tutorial
```

Comment:
Copying and pasting this block as is causes the following error:

```
Syntax error: failed at position 40 ('CREATE') (line 2, col 1):

CREATE DATABASE IF NOT EXISTS tutorial
CREATE TABLE tutorial.xyz (a UInt8, d Date) ENGINE = MergeTree() ORDER BY (a) PARTITION BY toYYYYMM(d);
INSERT INTO tutorial.xyz values(8,'2022-07-25');
SELECT * FROM tutorial.xyz;
```

Might be worth a quick note about the "create database" statement needing to be done separately from the other commands.

Reply:
Good point, I'll modify to be as separate commands.